android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -181,7 +183,7 @@ public synchronized void disconnect() {
       stopScanningDisplay();
     }
   }
-  private DiscoveryServiceConnection mDiscoveryServiceConnection = new DiscoveryServiceConnection();
+  private DiscoveryServiceConnection mDiscoveryServiceConnection;
```

Comment:
Could you move mDiscoveryServiceConnection so it's with the other member variables?